chrome-launcher.ts

```diff
@@ -123,7 +123,8 @@ export class Launcher {
     let flags = DEFAULT_FLAGS.concat([
       `--remote-debugging-port=${this.port}`,
       // Place Chrome profile in a custom location we'll rm -rf later
-      `--user-data-dir=${this.userDataDir}`
+      // If in WSL, we need to use the Windows format
+      `--user-data-dir=${toWinDirFormatIfNeeded(this.userDataDir)}`
```

Comment:
looks like the other uses of userDataDir don't need the path to be rewritten?

Reply:
No, this is only for WSL. The reason is that WSL access that path via `/mnt/c/...` but then we execute chrome in windows land and it doesn't recognize that path format at all so we need to convert it to `C:\...`